patterns/1-initial/change-the-developers-mindset.md

```diff
@@ -51,10 +51,11 @@ TBD
 
 ## Status
 
-Unproven
-
-Old status: Pattern Idea
+* Initial
+* Old status: Unproven
```

Comment:
Do we really need the old state here? If anyone is interested, the old status is accessible via the files history in git.

Reply:
We likely don't. I kept them around for stuff in level 1, so that I don't have to go through git logs when I review stuff.

I will keep this as is for now, and instead make a note to clean up that section when when we level up the pattern to level 2.